src/component/forcefield/HyperReducedHelper.h

```diff
@@ -173,7 +173,7 @@ class HyperReducedHelper : public virtual core::objectmodel::BaseObject
     void saveGieFile(unsigned int nbElements){
         if (d_prepareECSW.getValue())
         {
-            unsigned int numTest = this->getContext()->getTime()/this->getContext()->getDt();
+            unsigned int numTest = unsigned int(this->getContext()->getTime()/this->getContext()->getDt());
```

Comment:
This does not compile with GCC, as it thinks that it's supposed to call the ctor for int() and that "unsigned" is just a dangling keyword...
to replace with calls to unsigned directly, or size_t, uint32_t etc.